client/src/Register.js

```diff
@@ -75,6 +90,13 @@ function Register() {
                   <input type="password" class={classNames('form-control', {'is-invalid': error?.errorsFor?.('password')})} id="password" name="password" onChange={onChange} value={user.password} />
                   {error?.errorMessagesHTMLFor?.('password')}
                 </div>
+                <div className = "mb-3">
+                  <label className="form-label" htmlFor="userType">Select one of the following:</label>
+                  <select class="form-select" aria-label="Default select example" onChange={onChange} value={userType}>
```

Comment:
instead of having onChange call the existing onChange function above, try putting this in:

```
... onChange={(event) => setUserType(event.target.value)} ...
```

Reply:
You could also have created a new named function (i.e. onChangeUserType(event)) that did the same thing, but it would basically only have the one setUserType(event.target.value) call in it, so I think it's ok in this case to embed an anonymous function directly in the prop assignment.